src/backend/marsha/core/api/live_registration.py

```diff
@@ -118,6 +121,39 @@ def get_throttles(self):
 
         return [throttle() for throttle in throttle_class]
 
+    def create(self, request, *args, **kwargs):
+        """Create registration and send email"""
+        serializer = self.get_serializer(data=request.data)
+        if serializer.is_valid(raise_exception=True):
+            self.perform_create(serializer)
+            headers = self.get_success_headers(serializer.data)
+            translations = {
+                "registration_block2_title": _("mail_registration_block2_title"),
+                "registration_block2_paragraph": _(
+                    "mail_registration_block2_paragraph"
+                ),
+                "registration_extra_text": _("mail_registration_extra_text"),
+                "registration_label": _("mail_registration_label"),
+                "registration_link": _("mail_registration_link"),
+                "registration_success": _("mail_registration_success"),
```

Comment:
why not put this in the template?

Reply:
I don't think it would be possible in the text file? I'll give it a try

Reply:
The text file can be a template just like Html?

Reply:
Actually, with a bit of 
elbow grease, it works, I thought the i18n include would block the process of converting mjml into HTML, but placed at the right place, it works. And I didn't know but you can use translations in text files.